src/assets/stylesheets/utilities/styleguide.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/assets/stylesheets/global';
+
+@media #{$small-only} {
+  .step-progress-wrapper {
+    padding: 16px;
+    background-color: $brand-1;
```

Comment:
I think that this progress bar should have a transparent background -- but like we discussed, let's just leave it and move forward.